platform/osx/src/MGLMapView.mm

```diff
@@ -375,7 +375,6 @@ - (void)installAttributionView {
 
 /// Adds gesture recognizers for manipulating the viewport and selecting annotations.
 - (void)installGestureRecognizers {
-    self.acceptsTouchEvents = YES;
```

Comment:
I just tried out this one change (without the rest of the changes in this PR), and all the gestures continue to work fine. In fact, performance improves dramatically with this one change. I think `acceptsTouchEvents` is about raw touch events only, but MGLMapView uses gesture recognizers, which are unaffected by this property.